packages/slidev/node/syntax/transform/twoslash-conditional.ts

```diff
@@ -0,0 +1,17 @@
+import type { ShikiTransformer } from 'shiki'
+
+/**
+ * Custom Twoslash transformer that replaces :shown='true' with conditional logic
+ * to prevent poppers from being shown during preload
+ */
+export function transformerTwoslashConditional(): ShikiTransformer {
```

Comment:
We should use the build-time flag rather than a custom codemod (which is unstable), like:

https://github.com/slidevjs/slidev/blob/6750606dd1b7446ccb8de5030f913d3141f1e451/packages/slidev/node/options.ts#L119-L130